### PR TITLE
Set window size on startup

### PR DESCRIPTION
**Pull request by j-piecuch**
The `setwinsize` program moves the cursor to the bottom-right corner of the screen, reads the position of the cursor, and sets the terminal window size based on that.
The program is run only on startup, so if the actual window size changes at runtime, it won't be detected, but the user can update the screen size by invoking the `setwinsize` command manually.

### CODE REVIEW

File: bin/setwinsize/setwinsize.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  if (tcgetattr(term_fd, &old))
+    return err;
+  t = old;
+  cfmakeraw(&t);
+  if (tcsetattr(term_fd, TCSAFLUSH, &t))
+    return err;
+  /* Save current cursor position. */
+  fputs(save_cursor, termfile);
+  /* Move the cursor to the bottom right corner of the screen. */
+  const char *buf = tiparm(cursor_address, 1000, 1000);
```
Comment (pj1031999) [inline bin/setwinsize/setwinsize.c:32]: What does 1000 mean?

Comment (j-piecuch) [reply bin/setwinsize/setwinsize.c]: They are the row and column numbers that we're moving the cursor to. I have added more explanatory comments.

File: bin/setwinsize/setwinsize.c [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+int main(void) {
+  int err = 1;
+  struct termios old, t;
+  FILE *termfile;
+  /* Open controlling terminal. */
+  if ((termfile = fopen("/dev/tty", "r+")) == NULL)
+    return err;
+  int term_fd = fileno(termfile);
+  if (setupterm(NULL, term_fd, NULL))
```
Comment (hadarai) [inline bin/setwinsize/setwinsize.c:18]: I think you should add a comment about that that check also

**Review by xThaid — APPROVED** [review]

File: bin/ksh/config.h [hunk 3]
```diff
@@ -229,7 +229,7 @@
 #define POSIXLY_CORRECT 1
 
 /* Specify default $ENV? */
-#define DEFAULT_ENV	"$HOME/.kshrc"
+#define DEFAULT_ENV	"/etc/.kshrc"
```
Comment (cahirwpz) [inline bin/ksh/config.h:232]: Please mark `DEFAULT_ENV` with comment that we should restore it to original state when user home directories are available for write.

**Review by cahirwpz — APPROVED** [review]

**Pull request merged**